### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CaptainShadows**
Just a little clean up of the Readme to make the whole 1.5.2 - 1.6.2 thing easier to see. The whole letter stuff is because github is doing lists very weirdly. Anyhow just thought it looked a bit better

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +106,7 @@ not
 ```java
 public static void foo()
 {
-	return bar;
+    return bar;
```
Comment (CaptainShadows) [inline README.md:109]: no clue why that happend

File: README.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
 #Minecraft ID Resolver - A modpack creators best friend!
-
 Welcome! I have created a tool which I believe both normal users and modpack creators will be able to utilize. It is written in Java and therefore will work on every OS and is designed to find and fix Minecraft ID conflicts.
 
-#Usage
```
Comment (SS111) [inline README.md:5]: Don't change the titles (Usage, Building from source, Contributing, etc.) and I'll accept the pull.

Comment (CaptainShadows) [inline README.md:5]: Well.... I didn't really changed them, just made them a single hash smaller.

Comment (SS111) [inline README.md:5]: I'm aware. I'd rather them stay the same size. I do like what you did with the list, though. Fix the titles and I'll accept the pull.

**Pull request closed without merging**